### PR TITLE
Fix #21 update readme to include assets install

### LINKED ISSUE #21: Empty public folder of asset files
We should also then add further instructions to the readme of how to generate them

### PR DESCRIPTION
**Pull request by Douglasdc3**
Fixes #21 

* Add information on how to install assets
* Add yarn as default give hint about npm replacement

### CODE REVIEW

**Comment by Chinese1904** [comment]
Great, could you actually remove the assets inside the public folder as well? Those won't be needed anymore then.

File: readme.md [hunk 1]
```diff
@@ -8,15 +8,18 @@ This is an open source forum that was built and maintained at Laracasts.com.
...
 ```bash
 git clone git@github.com:JeffreyWay/council.git
-cd council && composer install
+cd council && composer install && yarn install
 mv .env.example .env
 php artisan key:generate
+yarn run dev
 ```
 
+> You can also use npm instead of yarn simply replace `yarn` with `npm`
```
Comment (Chinese1904) [inline readme.md]: > You can also use npm instead of yarn**;** simply replace `yarn` with `npm`


I would add a semicolon there

**Comment by desertcrystal** [comment]
Won't removing the static assets make installs more time consuming and complex, increasing the chance for errors. NPM/yarn should be optional.

**Comment by Chinese1904** [comment]
Adam Wathan's quote to a [question I asked](https://laracasts.com/discuss/channels/forge/should-i-run-npm-run-prod-locally-or-on-server) a while back:

> I personally do it on the server these days, using yarn instead of npm. Both have lock files now so it's not as dangerous as it used to be, and doing it on the server is nice because you don't have to remember to commit a production build of your assets before deploying.
> 
> I'd say if you are comfortable running composer install on your server (vs. doing it on a build server and rsyncing it over or similar), installing your npm dependencies on the server is fine.

**Comment by bambamboole** [comment]
and additionaly these files blow up the git repository. On each commit where its build new, its about 20000 new lines of code in the git repository.

**Comment by Douglasdc3** [comment]
@JeffreyWay updated! I agree stick to simple NPM approach

**Comment by JeffreyWay** [comment]
Thanks!

I also agree that we can switch to not including compiled files in git.

**Pull request merged**